### PR TITLE
Improves slip component docs

### PR DESCRIPTION
**Pull request by optimumtact**
Also fixes some comment headers from melbert

rendered

https://www.oranges.net.nz/~oranges/dmdoc/datum/component/slippery.html

### CODE REVIEW

**Comment by optimumtact** [comment]
I'll fix the extra change

**Review by Cyberboss — APPROVED** [review]

**Review by Cyberboss — APPROVED** [review]

File: code/datums/components/slippery.dm [hunk 1]
```diff
@@ -115,11 +145,12 @@
...
+/**
  * The proc that does the sliping. Invokes the slip callback we have set.
  *
- * source - the source of the signal
- * AM - the atom/movable that is being slipped.
+ * Arguments
+ * * source - the source of the signal
+ * * AM - the atom/movable that is being slipped.
  */
 /datum/component/slippery/proc/Slip(datum/source, atom/movable/arrived, atom/old_loc, list/atom/old_locs)
```
Comment (san7890) [inline code/datums/components/slippery.dm:155]:
```suggestion
 * * arrived - the atom/movable that is being slipped.
 */
/datum/component/slippery/proc/Slip(datum/source, atom/movable/arrived, atom/old_loc, list/atom/old_locs)
```

if we're going to document the args then the names should line up

File: code/datums/components/slippery.dm [hunk 2]
```diff
@@ -185,12 +219,13 @@
...
  * The slip proc, but for equipped items.
  * Slips the person who crossed us if we're lying down and unbuckled.
  *
- * source - the source of the signal
- * AM - the atom/movable that slipped on us.
+ * Arguments:
+ * * source - the source of the signal
+ * * AM - the atom/movable that slipped on us.
  */
 /datum/component/slippery/proc/Slip_on_wearer(datum/source, atom/movable/arrived, atom/old_loc, list/atom/old_locs)
```
Comment (san7890) [inline code/datums/components/slippery.dm:230]:
```suggestion
 * * arrived - the atom/movable that slipped on us.
 */
/datum/component/slippery/proc/Slip_on_wearer(datum/source, atom/movable/arrived, atom/old_loc, list/atom/old_locs)
```

ditto

**Review by san7890 — COMMENTED** [review]
naming nits

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**